lib/stack_master/parameter_resolvers/acm_certificate.rb

```diff
@@ -0,0 +1,37 @@
+module StackMaster
+  module ParameterResolvers
+    class AcmCertificate < Resolver
+      CertificateNotFound = Class.new(StandardError)
+
+      def initialize(config, stack_definition)
+        @config = config
+        @stack_definition = stack_definition
+      end
+
+      def resolve(domain_name)
+        cert_arn = find_cert_arn_by_domain_name(domain_name)
+        raise CertificateNotFound, "Could not find certificate #{domain_name} in #{@stack_definition.region}" unless cert_arn
+        cert_arn
+      end
+
+      private
+
+      def all_certs
+        certs = []
+        next_token = nil
+        client = Aws::ACM::Client.new(region: @stack_definition.region)
+        loop do
+          resp = client.list_certificates(next_token: next_token)
```

Comment:
Should we filter for only `ISSUED` certificates?

Reply:
Good call. Done.